draft-ietf-quic-recovery.md

```diff
@@ -291,6 +291,13 @@ are received within an RTT.  As suggested in {{?RFC6298}}, doing so might result
 in inadequate history in smoothed_rtt and rttvar.  Ensuring that RTT estimates
 retain sufficient history is an open research question.
 
+### Retransmitted ACK frames
+
+Section 13.3 of {{QUIC-TRANSPORT}} recommends sending the most recent set of
+acknowledgements in ACK frames, but if ACK frames are retransmitted, an inflated
```

Comment:
```suggestion
acknowledgements in ACK frames, but if ACK frames are retransmitted without adjusting the Ack Delay, an inflated
```

Reply:
I originally wrote that, but it's likely to be inflated anyway, particularly if max_ack_delay is < RTT.